packages/react-query-devtools/eslint.config.js

```diff
@@ -1,24 +1,22 @@
 // @ts-check
 
 import pluginReact from '@eslint-react/eslint-plugin'
-import * as reactHooks from 'eslint-plugin-react-hooks'
+import reactHooks from 'eslint-plugin-react-hooks'
 import rootConfig from './root.eslint.config.js'
 
 export default [
   ...rootConfig,
-  reactHooks.configs.recommended,
+  // @ts-expect-error wtf
```

Comment:
drive-by comment as I saw your posts on Bluesky...

if you hadn't already come across it, thought it'd be helpful to point to https://github.com/facebook/react/issues/34745, which explains (some of?) the type issues with integrating the plugin's config (and the expect error overrides you needed).

Issue was closed but the fix hasn't been published yet. You may have a better time with integration once it is 😄